debian/control

```diff
@@ -0,0 +1,31 @@
+Source: nntrainer
+Section: libs
+Priority: optional
+Maintainer: Jijoong Moon <jijoong.moon@samsung.com>
+Build-Depends: gcc-9 | gcc-8 | gcc-7 | gcc-6 | gcc-5 (>=5.4),
+ ninja-build, meson (>=0.50), debhelper (>=9),
+ libblas-dev, libiniparser-dev
+Standards-Version: 0.0.1
```

Comment:
Standards-Version is not for representing the version of this package. Please refer https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-standards-version

Reply:
I see. Thanks.!